care/facility/api/serializers/patient.py

```diff
@@ -228,3 +231,14 @@ def create(self, validated_data):
     def save(self, **kwargs):
         self.instance.facility = self.validated_data["facility"]
         self.instance.save()
+
+
+class PatientField(serializers.Field):
+    def to_representation(self, patient):
+        return patient.id_hash
+
+    def to_internal_value(self, id_hash):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5408178)